_episodes/09-cmorization.md

```diff
@@ -0,0 +1,1053 @@
+some things to make sure the dataset name can be correctly created, and
+ultimately the ESMValTool knows how to look for the new dataset.
+
+Therefore it is necessary to create a configuration file for the new dataset.
+This configuration file needs to be stored in the
+following folder:
+
+```bash
+ESMValTool/esmvaltool/cmorizers/obs/cmor_config/
+```
+It is important to note that the name of the configuration file has to be
+identical to the name of the dataset. For our example the configuration file,
+traditionally written in the ``yaml`` format, therefore must be called
+``FLUXCOM.yml``.
+
+Let's have a closer look what that configuration file needs to contain:
+- information about the filenames of the raw observation files (stored in the
+"RAWOBS" folder)
+- information about "global attributes" for the netCDF file that will be
+created (both for the name of the file and the metadata that the file needs to
+contain)
+- information about the variables that need to be cmorized
+
+> ## Let's create the configuration file for the "FLUXCOM" dataset
+>
+> Here is the skeleton of the "FLUXCOM" configuration file as it exists in
+> the ESMValTool framework. Try to fill in all missing pieces of information
+> for this configuration file that are marked with ``???``.
+>
+> ```yaml
+> ---
+> # Filename
+> filename: ???
+>
+> # Common global attributes for Cmorizer output
+> attributes:
+>   dataset_id: ???
+>   version: ???
+>   tier: ???
+>   modeling_realm: ???
+>   project_id: ???
+>   source: ???
+>   reference: ???
+>   comment: ???
+>
+> # Variables to cmorize
+> variables:
+>   ???:
+>     mip: ???
+> ```
+>
+> > ## Answers
+> >
+> > The configuration file for the "FLUXCOM" dataset with all necessary pieces
+> > of information looks  like this:
+> >
+> > ```yaml
+> > ---
+> > # Filename
+> > filename: 'GPP.ANN.CRUNCEPv6.monthly.*.nc'
+> >
+> > # Common global attributes for Cmorizer output
+> > attributes:
+> >   dataset_id: FLUXCOM
+> >   version: 'ANN-v1'
+> >   tier: 3
+> >   modeling_realm: reanaly
+> >   project_id: OBS
+> >   source: 'http://www.bgc-jena.mpg.de/geodb/BGI/Home'
+> >   reference: 'fluxcom'
+> >   comment: ''
+> >
+> > # Variables to cmorize
+> > variables:
+> >   gpp:
+> >     mip: Lmon
+> > ```
+> >
+> > The original configuration file for the "FLUXCOM" dataset can be
+> > found here:
+> > [FLUXCOM.yml]
+> > (https://github.com/ESMValGroup/ESMValTool/blob/master/esmvaltool/cmorizers/obs/cmor_config/FLUXCOM.yml)
+> >
+> > Note the attribute "reference" here: it should include a ``doi`` related
+> > to the dataset. For more information on how to add references to the
+> > ``reference`` section of the configuration file, see the section in the
+> > documentation about this: [adding references]
+> > (https://docs.esmvaltool.org/en/latest/community/diagnostic.html#adding-references)
+> >
+> > If a single dataset has more than one reference, it is possible to add
+> > tags as a list e.g. ``reference: ['tag1', 'tag2']``.
```

Comment:
This information is already mentioned in the documentation and it beyond the scope of this episode. Let's remove it to make the lesson concise.

Reply:
Done